### ISSUE TITLE
Can't build rusk-wallet-0.5.3 from source on macos + m1

### ISSUE THREAD

**Issue by mfreeman451** [dusk-network/rusk#684]
**Description**

After using 'brew install rust' on an m1 and then running 'make run' the build fails

**Reproduction**

```
brew install rust

make run
```

```
  Compiling parking_lot_core v0.9.2
   Compiling nodrop v0.1.14
   Compiling smallvec v1.8.0
   Compiling cfg-if v1.0.0
   Compiling scopeguard v1.1.0
   Compiling constant_time_eq v0.1.5
error[E0463]: can't find crate for `core`
  |
  = note: the `wasm32-unknown-unknown` target may not be installed
  = help: consider downloading the target with `rustup target add wasm32-unknown-unknown`

error[E0463]: can't find crate for `compiler_builtins`

error: cannot find macro `assert` in this scope
 --> .cargo/registry/src/github.com-1ecc6299db9ec823/constant_time_eq-0.1.5/src/lib.rs:6:5
  |
6 |     assert!(a.len() == b.len());
  |     ^^^^^^

error: requires `sized` lang_item

For more information about this error, try `rustc --explain E0463`.
error: could not compile `constant_time_eq` due to 4 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed
make[2]: *** [wasm] Error 101
make[1]: *** [stake/.] Error 2
make: *** [wasm] Error 2
```

**Platform**

 21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_ARM64_T8101 arm64

**Additional context**

Source tar.gz was downloaded from https://github.com/dusk-network/rusk/releases/tag/v0.5.3-wallet

**Comment by ZER0** [comment]
Hi, sorry for my late reply.
The solution of your problem is said in the error message:

> help: consider downloading the target with `rustup target add wasm32-unknown-unknown`

You need to run `rustup target add wasm32-unknown-unknown` after you've installed rust so it can install the web assembly target for your machine, and then you can run `make run`. Notice that this operation has to be done only once, then you can compile any time you want without calling `rustup`.

Notice also that the wallet has now it's own repo, and it was just released version 0.7: https://github.com/dusk-network/wallet-cli/

**Comment by ureeves** [comment]
Closing as stale. Please refer to the [wallet repository](https://github.com/dusk-network/wallet-cli)
